[Flutter] configurations examples for AppKit (#244)
reown-com/reown-docs: docs/appkit/flutter/core/link-mode.mdx

By enabling it, the wallet and dapp will communicate through declared Universal Links on iOS and/or App Links on Android **even without an internet connection.**

:::caution important note

> In order to fully works, Link Mode requires you to enable also [One-Click Auth + Sign In With Ethereum](./siwe.mdx) feature.

Reviewer: ```suggestion
For Link Mode to work fully, you also need to enable the [One-Click Auth + Sign In With Ethereum](./siwe.mdx) feature.
```

Minor grammar fix
